code-postprocessing/cocopp/compall/pprldmany.py

```diff
@@ -35,6 +35,7 @@
 from pdb import set_trace
 import numpy as np
 import matplotlib.pyplot as plt
+from matplotlib.ticker import (FixedLocator, FuncFormatter, NullFormatter)
```

Comment:
minor comment: using braces here doesn't look like python coding style.

Reply:
Busted! I copied the code from [here](https://matplotlib.org/gallery/ticks_and_spines/major_minor_demo.html#major-minor-demo) and adapted it. I'll make a note for future work. Thanks for the comment.